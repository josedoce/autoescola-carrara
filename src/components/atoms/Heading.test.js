import React from 'react';
import { render } from '../../test-utils';

import Heading from './Heading';

test('match snapshot', ()=>{
  const {asFragment} = render(
    <Heading>titulo</Heading>
  );
  expect(asFragment()).toMatchSnapshot();
})
