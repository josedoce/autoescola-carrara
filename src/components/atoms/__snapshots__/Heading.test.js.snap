// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ NDMox"
  >
    titulo
  </div>
</DocumentFragment>
`;
